libs/events/src/types/orders.ts

```diff
@@ -21,7 +21,9 @@ export type OnPostedOrderPayload = {
   isEthFlow?: boolean
 }
 
-export type OnFulfilledOrderPayload = BaseOrderPayload
+export type OnFulfilledOrderPayload = BaseOrderPayload & {
+  bridgeOrder?: BridgeOrderDataSerialized
```

Comment:
SInce the events are being forwarded to widget integrator, they have to be serialized. See `CowEventsUpdater`